src/controllers/usersController.js

```diff
@@ -0,0 +1,41 @@
+import UsersService from '../services/usersService';
+import Response from '../utils/response';
+
+const { success, badRequest } = Response;
+
+/**
+ * Class for interacting with User profile information
+ */
+export default class UsersController {
+/**
+ * @param {object} user - user auth token payload
+ * @param {object} res - response object
+ * @return {object} user - return object containing status and data
+ */
+  static async getProfile({ user }, res) {
+    try {
+      const { id } = user;
+      const userData = await UsersService.getProfile(id);
+
+      success(res, userData);
+    } catch ({ message: error }) {
+      badRequest(res, error);
+    }
+  }
+
+  /**
+ * @param {object} user - user auth token payload
+ * @param {object} res - response object
+ * @return {object} user - return object containing status and data
+ */
+  static async updateProfile({ body, user }, res) {
+    try {
+      const { id } = user;
+      const updatedData = await UsersService.updateProfile(id, body);
+
+      success(res, updatedData);
+    } catch ({ message: error }) {
+      badRequest(res, error);
+    }
+  }
+}
```

Comment:
We have `AuthController`, please ultilize that instead of creating your own controller.

Reply:
Ok, noted

Reply:
Do I also utilize `AuthService` as well?

Reply:
Yes Please..

Reply:
Alright